hcipy/field/util.py

```diff
@@ -359,6 +359,8 @@ def subsample_field(field, subsampling, new_grid=None, statistic='mean'):
 		* 'sum': compute the sum of values for points within each superpixel. This is identical to a weighted histogram.
 		* 'min': compute the minimum of values for points within each superpixel.
 		* 'max': compute the maximum of values for point within each superpixel.
+		* 'median': compute the median of values for point within each superpixel.
+		* 'nanmedian': compute the median of values for point within each superpixel.
```

Comment:
Why add a nanmedian as well? Also, modify the documentation for this if you want to keep it.

Reply:
I use hcipy for some data reduction pipelines and I needed the nanmedian for something.

Reply:
Okay, then add in the documentation that this ignores NaNs inside the superpixel.

Reply:
You marked as resolved, but I see no mention of ignoring nans in the documentation on this line.